packages/angular_devkit/build_ng_packagr/package.json

```diff
@@ -11,13 +11,19 @@
     "rxjs": "6.5.3"
   },
   "peerDependencies": {
-    "ng-packagr": "^9.0.0-rc.0"
+    "ng-packagr": "^9.0.0-rc.0",
+    "tsickle": "~0.37.1"
```

Comment:
I was under the impression that the fix for this was the include the dependency in the packageGroup.

Reply:
Yeah that was the original idea, but then Charles brough the optional peer dep up that both ensures it is updated, and ensures that if it is present on install (outside of update) it should be the right version. Do you think there are problems with this approach?

Reply:
Not at all, I was just curios to know why the change :)